README.md

```diff
@@ -28,56 +28,70 @@ familiar with this ask a dev or an admin to help you.
 
 - Click on the pencil icon to edit the page. That will take you to the corresponding page in github.
 
-![Screenshot of a page on info.arxiv.org and the location of the edit icon](/non-info/02-help-info-page.png)
+<kbd>![Screenshot of a page on info.arxiv.org and the location of the edit icon](/non-info/02-help-info-page.png)</kbd>
 
 - On the page you want to edit in github, click the pencil to edit the page contents right in github.
 
-![Screenshot of a page in Github and the location of the edit icon](/non-info/01-help-github1a.png)
+<kbd>![Screenshot of a page in Github and the location of the edit icon](/non-info/01-help-github1a.png)</kbd>
 
 - After you have made changes, click preview to see how it will look when completed. 
-**Please note:** There are slight differences between the markdown we use and what github uses.
+**Please note:** Github uses "Github flavored markdown" which is different from the markdown that we use to create our documentation. There may be small variation in how the markdown appears due to these differences.
 
-![Screenshot of the location of the preview button on the edited page in Github](/non-info/03-help-preview-github.png)
+<kbd>![Screenshot of the location of the preview button on the edited page in Github](/non-info/03-help-preview-github.png)</kbd>
 
 - Once you are happy with your changes you can save by clicking on the blue "Commit changes" button in the upper right hand corner.
 Commit is the term github and git use for saving.
 
-![Screenshot of the location of the commit button on the edited page in Github](/non-info/04-help-commit-github.png)
+<kbd>![Screenshot of the location of the commit button on the edited page in Github](/non-info/04-help-commit-github.png)</kbd>
 
 - Add a brief note in the "Commit message" area summarizing your edit. You may add longer
 note in the `Extended description` area if you have more to say about your changes.
 
-- Make sure you have an email associated with your commits.
+- Make sure you have your email associated with your commits.
 
 - Create a "new branch for this request and start a pull request" will be selected
-- Name your branch after your JIRA ticket to link it. Ex `arxivce-1503-update-arxiv-docs-readme`
+- Name your branch after your JIRA ticket to link it. Ex `arxivce-1503-update-arxiv-docs-readme`.
 
-![Screenshot of how to create a new branch for the edits made in Github](/non-info/05-help-newBranch.png)
+<kbd>![Screenshot of how to create a new branch for the edits made in Github](/non-info/05-help-newBranch.png)</kbd>
 
-- **Please note:** The above step will only put your changes on develop and they will not be viewable on info.arxiv.org. At this point you have only saved the edit in github.
+- **Please note:** The above step will only put your changes on a branch on develop and they will not be viewable on info.arxiv.org. At this point you have only saved the edit in github.
```

Comment:
a branch on develop isnt really correct. develop is a branch, your changes will be on a different branch

Reply:
Got it. I moved this comment after one merges their branch into develop. It makes more sense there.